src/cmdline.rs

```diff
@@ -186,12 +186,12 @@ pub struct InstallConfig {
     pub copy_network: bool,
     /// For use with -n.
     #[structopt(long, value_name = "path")]
-    #[structopt(default_value = "/etc/NetworkManager/system-connections/")]
+    #[structopt(default_value)]
```

Comment:
Minor/optional: could unify these now for consistency.

Reply:
Yup, good call.  Also added a commit to drop `structopt(verbatim_doc_comment)` from that field for simplicity.